src/iop/basecurve.c

```diff
@@ -317,13 +316,13 @@ static void set_presets(dt_iop_module_so_t *self, const basecurve_preset_t *pres
       tmp.exposure_bias = 1.0f;
     }
     // add the preset.
-    dt_gui_presets_add_generic(_(presets[k].name), self->op, self->version(),
```

Comment:
likewise. I'll stop read from here. Please remove all the reformatting out of the actual patch doing the real work.

thanks.

Reply:
Reformatting is a result of clang-format, which has reformatted the code according to the darktable formatting style. I was required to integrate clang-format into git by the darktable developers in the past. I have *not* applied these reformatting by hand.

Reply:
That's not what I was told and what I heard. We have a reformatting script that we do run from time to time. Having all those reformatting into a PR is just noise and makes it hard to read. As-is I won't approve it anyway as I cannot see exactly what has been really changed. Too much work, sorry.

Reply:
And to be clear, I'm not against the reformatting as long as it is on a separate commit to ease the review.